### PR TITLE
Updated spelling and other syntax adjustements.

### PR DESCRIPTION
**Pull request by AlexSkrypnyk**

### CODE REVIEW

File: scripts/deploy/govcms-cache-rebuild [hunk 1]
```diff
@@ -7,11 +7,14 @@ set -euo pipefail
...
 # Allow a cache-rebuild to be toggled at deploy time.
 #
+
 LAGOON_ENVIRONMENT_TYPE=${LAGOON_ENVIRONMENT_TYPE:-production}
 
+GOVCMS_SKIP_CACHE_REBUILD=${GOVCMS_SKIP_CACHE_REBUILD:-}
+
 echo "GovCMS Deploy :: Cache rebuild"
 
-if [ -z "${GOVCMS_SKIP_CACHE_REBUILD-}" ]; then
```
Comment (AlexSkrypnyk) [inline scripts/deploy/govcms-cache-rebuild:14]: this was not working

File: scripts/deploy/govcms-config-backup [hunk 2]
```diff
@@ -32,4 +32,5 @@ drush config:export sync -y --destination "$GOVCMS_BACKUP_DIR/config"
 tar -czf "$GOVCMS_BACKUP_DIR/$GOVCMS_CONFIG_BACKUP.tar.gz" -C "$GOVCMS_BACKUP_DIR/config" --remove-files
 
 echo "[info]: Saved to $GOVCMS_BACKUP_DIR/$GOVCMS_CONFIG_BACKUP.tar.gz"
+
```
Comment (AlexSkrypnyk) [inline scripts/deploy/govcms-config-backup:35]: here and below - success message is a last code line and has exactly one empty line above it.

File: scripts/deploy/govcms-config-import [hunk 3]
```diff
@@ -3,7 +3,7 @@ IFS=$'\n\t'
 set -euo pipefail
 
 #
-# GovCMS configuration import
+# GovCMS configuration import.
```
Comment (AlexSkrypnyk) [inline scripts/deploy/govcms-config-import:6]: here and below - every sentence is terminated by dot.

File: scripts/deploy/govcms-pre-deploy [hunk 4]
```diff
@@ -1,26 +1,25 @@
 #!/usr/bin/env bash
-IFS=$'\n'
+IFS=$'\n\t'
```
Comment (AlexSkrypnyk) [inline scripts/deploy/govcms-pre-deploy:2]: here and below - using consistent `IFS` across scripts - unless this was inconsistent intentionally?

File: scripts/deploy/govcms-update-site-alias [hunk 5]
```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
```
Comment (AlexSkrypnyk) [inline scripts/deploy/govcms-update-site-alias:1]: renamed to be hyphenated

File: scripts/deploy/govcms-update-site-alias [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+# @deprecated 1.0
+#
+# This is used in the legacy projects that have not been updated with the latest
+# scaffold.
+#
+
+echo "GovCMS Deploy :: Update site alias"
+
+if [ ! -f /app/drush/sites/govcms.site.yml ]; then
+  echo "[skip]: Site alias file does not exist."
```
Comment (AlexSkrypnyk) [inline scripts/deploy/govcms-update-site-alias:20]: added verbose skip to be consistent with other scripts

File: scripts/deploy/govcms-pre-deploy (hunk 4 above)
Comment (simesy) [reply scripts/deploy/govcms-pre-deploy:2]: Not intentional. looks good

Comment (steveworley) [reply scripts/deploy/govcms-pre-deploy:2]: That was intentional for this script, it allows us to add multiple commands via the env var separated by a space.

Comment (simesy) [reply scripts/deploy/govcms-pre-deploy]: ok great.. possibly should comment or modify it when it needs to be modified? Setting a default is to ensure unexpected events.

**Pull request merged**